ocs_ci/ocs/ocp.py

```diff
@@ -43,14 +43,18 @@ def kind(self):
     def namespace(self):
         return self._namespace
 
-    def exec_oc_cmd(self, command):
+    def exec_oc_cmd(self, command, raw_output=False):
```

Comment:
1) I'd do the opposite - out_yaml_format=True for consistency - see below functions
and if you agree with that,
```python
 if not out_yaml_format:
   ....
```

Reply:
used same name